client/state/themes/actions.js

```diff
@@ -364,6 +368,49 @@ export function clearActivated( siteId ) {
 }
 
 /**
+ * Triggers a network request to install WordPress.com theme on Jetpack site.
+ * After installataion it switches page to the customizer
+ * Requires Jetpack 4.4
+ *
+ * @param  {String}   siteId       Jetpack Site ID
+ * @param  {String}   wpcomThemeId WP.com Theme ID
+ * @return {Function}              Action thunk
+ */
+export function tryAndCustomizeWpcomThemeOnJetpack( themeId, siteId ) {
+	//Add -wpcom suffix. This suffix tells the endpoint that we want to
+	//install WordPress.com theme. Without the suffix endpoint would look
+	//for theme in .org
+	const suffixedThemeId = themeId + '-wpcom';
+
+	return ( dispatch, getState ) => {
+		dispatch( {
+			type: THEME_TRY_AND_CUSTOMIZE_ON_JETPACK_REQUEST,
```

Comment:
>  I think we should finally introduce the installWpcomThemeOnJetpack action along with properly named action name constants, and compose both activateWpcomThemeOnJetpack and tryAndCustomizeWpcomThemeOnJetpack from it.

Yeah, I would suggest that now we have the two lists and install working, we can take the time to do this properly.